Test/pruning/MonomorphicSplit.bpl

```diff
@@ -0,0 +1,37 @@
+// RUN: %parallel-boogie /prune /errorTrace:0 /printPruned:"%t" "%s" > "%t"
+// RUN: %OutputCheck "%s" --file-to-check="%t-after-monomorphicSplit.bpl"
+
+// The following checks are a bit simplistic, but this is
+// on purpose to reduce brittleness. We assume there would now be two uses clauses
+// with one axiom each, and those axioms should not be conjunctions
```

Comment:
How can an axiom be a conjunction of monomorphized functions? The types do not match. Please rephrase.

The last sentence of this paragraph is equally mysterious.